relay/relay.go

```diff
@@ -204,7 +204,7 @@ func (r *Relay) process(ctx context.Context) error {
 		return err
 	}
 
-	if isNew || r.cfg.UUIDSuffix > 0 {
+	if isNew {
```

Comment:
if we don't remove it, an old upstream will enter then-branch because its UUIDSuffix is not zero (in most logic it will be set ti `MinUUIDSuffix` which is `1`)